doc/manual/01-introduction.en.md

```diff
@@ -0,0 +1,170 @@
+\newpage
+
+# Introduction
+
+[**pgmoneta**](https://github.com/pgmoneta/pgmoneta) is a backup / restore solution for [PostgreSQL](https://www.postgresql.org).
+
+**pgmoneta** is named after the Roman Goddess of Memory.
+
```

Comment:
We need something something about RPO and RTO in the Introduction, but we could take that later